megamek/src/megamek/client/ui/swing/TilesetManager.java

```diff
@@ -126,12 +128,18 @@
      * in the same hex).
      */
     private HashMap<Color, Image> ecmStaticImages = new HashMap<Color, Image>();
+    
+    // The TileManager is associated with a game passed down from the Boardview
+    // and must pass this on to the hextileset for it to register listeners
+    private IGame game;
 
     /**
      * Creates new TilesetManager
      */
-    public TilesetManager(BoardView1 bv) throws IOException {
+    public TilesetManager(BoardView1 bv, IGame g) throws IOException {
```

Comment:
I'd like to point out that BoardView already has a game object, so there's not a need to pass it around.

Reply:
OK, I have another solution. If there is some obviously better one, please do point me to it. I'm open towards at least a megamek-internal philosophy on that. I made boardview's game public and final and removed the pass to the Tilesetmanager.

Reply:
Is there something I should do because of the board validator?

Reply:
Yeah, if you click on the details for the Validate Boards check, you can look at which boards fail as a result of the changes made and address those.